docs/contributing/development_environment_examples/standalone.j2

```diff
@@ -27,7 +27,7 @@ network_config:
   domain: {{ dns_search_domains }}
   members:
     - type: interface
-      name: {{ neutron_public_interface_name }}
+      name: {{ edpm_network_config_interface_name }}
```

Comment:
This file is being used by TripleO Wallaby standalone deployment. AFAICT chanigng the name here would break the initial deployment.